### PR TITLE
S3 encryption at rest

### PR DESCRIPTION
**Pull request by giorgiosironi**
For https://github.com/elifesciences/elife-xpub/issues/704

### CODE REVIEW

**Comment by diversemix** [comment]
Are you happy to store the key alongside others? If we add this to the docs .... where shall we say it is stored?

File: projects/elife.yaml [hunk 1]
```diff
@@ -1975,12 +1976,10 @@ elife-xpub:
             device: /dev/sdh
             type: gp2
     aws-alt:
-        demo:
-            s3:
-                "{instance}-elife-xpub":
         staging:
```
Comment (diversemix) [inline projects/elife.yaml]: Is `prod` getting added in another PR?

**Comment by giorgiosironi** [comment]
> Are you happy to store the key alongside others?

It is stored in the KMS console which a few IT admins can access; the idea of KMS being it stores it for you rather than having to set up yet another Git repo and its permissions.

File: projects/elife.yaml (hunk 1 above)
Comment (giorgiosironi) [reply projects/elife.yaml]: This is the deletion of `demo` which is configured but not used; both `staging` and `prod` have been configured to use `encryption`, look at the next additions in this file.

**Comment by giorgiosironi** [comment]
Previous of [KMS in RDS encryption at rest](https://github.com/elifesciences/builder/pull/394#issuecomment-424545203)

**Pull request merged**